src/screens/TripDetailsScreen.tsx

```
@@ -91,16 +143,83 @@ const TripDetailsScreen: React.FC = () => {
-          <button className="button is-primary" onClick={handleSaveTrip}>
-            Save Trip
-          </button>
+      {trip && trip.team.length > 1 && (
+      <h2 className="subtitle">
+        <div className="columns">
+          <div className="column is-four-fifths">Payments</div>
+          <div className="column has-text-right">
+            <button
+              className="button is-primary mb-4"
+              onClick={() => setIsPaymentModalOpen(true)}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)